xla/hlo/builder/lib/math.cc

```diff
@@ -123,14 +124,21 @@ static absl::Status EnsureOperandIsRealFp(absl::string_view op_name,
   return absl::OkStatus();
 }
 
+XlaOp PredFalse(XlaBuilder* builder, const Shape& shape) {
+  Shape pred_shape = ShapeUtil::ChangeElementType(shape, PRED);
+  return ConstantLiteral(builder, Literal::CreateFromShape(pred_shape));
```

Comment:
Maybe instead use:

```
return Broadcast(ConstantR0WithType(builder, PRED, false));
```

Using broadcast of a scalar constant is preferred, and making the init value of the constant literal explicit is easier to understand.

Reply:
Done, thanks.